src/Bicep.Core/Navigation/SyntaxBaseExtensions.cs

```diff
@@ -78,6 +80,42 @@ public static string ToText(this SyntaxBase syntax, string indent="")
             document.Layout(sb, indent, System.Environment.NewLine);
             return sb.ToString();
         }
+
+        /// <summary>
+        /// Generate a string that represents this Syntax element optionally preserving formatting.
```

Comment:
>optionally preserving formatting [](http://example.com/codeflow?start=66&length=32)

Can you remove the "optionally preserving formatting" from the comment since it always preserves formatting?